aspnetcore/blazor/file-downloads.md

```diff
@@ -84,14 +84,12 @@ The `fileName` and object `url` are passed to `triggerFileDownload`, which perfo
 * Trigger the download via `click`.
 * Remove the anchor (`<a>`) element.
 
-At this point, the file download is triggered. It should be safe to revoke the temporary object URL by calling [`revokeObjectURL`](https://developer.mozilla.org/docs/Web/API/URL/revokeObjectURL) on the URL. **This is an important step to ensure memory isn't leaked on the client.**
-
+At this point, the file download is triggered, and it's generally safe to revoke the temporary object URL by calling [`revokeObjectURL`](https://developer.mozilla.org/docs/Web/API/URL/revokeObjectURL) on the URL. **This is an important step to ensure memory isn't leaked on the client.**
```

Comment:
I like your suggested wording:

`or strike the "generally" language in favor of just saying that the URL should be revoked with the API as indicated.`

As far as I can tell, immediately revoking the object url after starting the download / anchorElement.click() is sufficient as the browser gets a "copy of the object in the blob store" so the revoking the original URL is inconsequential.

https://bugzilla.mozilla.org/show_bug.cgi?id=1282407#c5

Reply:
```suggestion
At this point, the file download is triggered and then the temporary object URL is revoked by calling [`revokeObjectURL`](https://developer.mozilla.org/docs/Web/API/URL/revokeObjectURL) on the URL. **This is an important step to ensure memory isn't leaked on the client.**
```